.appveyor.yml

```diff
@@ -2,13 +2,30 @@ image: Visual Studio 2017
 
 configuration: Release
 
-before_build:
+install:
   - cmd: dotnet restore src\Chinchilla.sln
-  
+
+dotnet_csproj:
+  patch: true
+  file: 'src\**\*.csproj'
+  version: $(appveyor_build_version)
+  package_version: $(appveyor_build_version)
+
 build:
   project: src\Chinchilla.sln
-  
-test:
-  assemblies:
-    only:
-      - 'targets\specifications\**\*.Specifications.dll'
+  publish_nuget: true
+
+test_script:
+  - ps: .\build\run-tests.ps1
```

Comment:
`dotnet test` is not a first-class citizen of appveyor yet, which is why we need this PS script.